Buildings/Controls/OBC/CDL/Logical/Sources/TimeTable.mo

```diff
@@ -0,0 +1,235 @@
+  connect(intTru.y, intEquTru.u1)
+    annotation (Line(points={{-58,50},{-42,50}}, color={255,127,0}));
+  connect(intEquFal.y, or2.u1) annotation (Line(points={{-18,80},{0,80},{0,70},{
+          18,70}}, color={255,0,255}));
+  connect(intEquTru.y, or2.u2) annotation (Line(points={{-18,50},{0,50},{0,62},{
+          18,62}}, color={255,0,255}));
+  connect(or2.y, mulAnd.u)
+    annotation (Line(points={{42,70},{48,70}}, color={255,0,255}));
+  connect(intEquTru.y, y) annotation (Line(points={{-18,50},{0,50},{0,0},{140,0}},
+        color={255,0,255}));
+  connect(realToInteger.y, intToRea.u) annotation (Line(points={{-38,0},{-30,0},
+          {-30,-20},{-22,-20}}, color={255,127,0}));
+  connect(intToRea.y, add2.u1) annotation (Line(points={{2,-20},{10,-20},{10,-44},
+          {18,-44}}, color={0,0,127}));
+  connect(tab.y, add2.u2) annotation (Line(points={{-79,0},{-70,0},{-70,-56},{18,
+          -56}}, color={0,0,127}));
+  connect(add2.y, abs1.u) annotation (Line(points={{42,-50},{50,-50},{50,-70},{-70,
+          -70},{-70,-90},{-62,-90}}, color={0,0,127}));
+  connect(abs1.y, greThr.u)
+    annotation (Line(points={{-38,-90},{-22,-90}}, color={0,0,127}));
+  connect(greThr.y, mulOr1.u)
+    annotation (Line(points={{2,-90},{18,-90},{18,-90}}, color={255,0,255}));
+  connect(mulOr1.y, not1.u)
+    annotation (Line(points={{42,-90},{58,-90}}, color={255,0,255}));
+  connect(not1.y, and2.u2) annotation (Line(points={{82,-90},{90,-90},{90,-20},{
+          50,-20},{50,22},{58,22}}, color={255,0,255}));
+  connect(mulAnd.y, and2.u1) annotation (Line(points={{72,70},{80,70},{80,50},{50,
+          50},{50,30},{58,30}}, color={255,0,255}));
+  connect(and2.y, assMes1.u) annotation (Line(points={{82,30},{90,30},{90,70},{98,
+          70}}, color={255,0,255}));
+  annotation (
+defaultComponentName = "intTimTab",
+Documentation(info="<html>
+<p>
+Block that outputs <code>true</code>/<code>false</code> time table values.
+</p>
+<p>
+The block takes as a parameter a time table of a format:
+</p>
+<pre>
+table = [ 0*3600, 0;
+          6*3600, 1;
+          6*3600, 0;
+         18*3600, 1;
+         18*3600, 1;
+         24*3600, 1];
+</pre>
+<p>
+where the first column is time, and the remaining column(s) are the table values.
+The time column contains <code>Real</code> values that are in units of seconds when <code>timeScale</code> equals <code>1</code>.
+<code>timeScale</code> may be used to scale the time values, such that for <code>timeScale = 3600</code> the values 
+in the first column of the table are interpreted as hours.
+<p>
+Any number of columns can be specified.
+</p>
+<p>
+The values in all columns apart from the first column must equal either <code>0</code> or <code>1</code>, 
+to represent <code>false</code> or <code>true</code>, respectively, otherwise a warning is issued.
+</p>
+<p>
+The parameter <code>smoothness</code> determines how the table values
+are interpolated. The following setting is implemented:
+</p>
```

Comment:
There is no smoothness parameter (and it makes not sense to use smoothness for a boolean signal).

Reply:
I can protect it, yet we need to define how the values between the defined points in time are determined.

Reply:
You could set it directly in the instance declaration or as a protected parameter; but the info section should not mention it. Typical users may just read the info section, so explaining smoothness will be confusing if there is no such option (and there should not be such an option for the user).

Reply:
That sounds good to me.